graph/client.py

```diff
@@ -107,10 +120,19 @@ def prepare_graphs(self, rules):
     def open_web_browser(self):
         if not self.off_webbrowser:
             try:
-                webbrowser.get('firefox').open_new_tab(
-                    'html_interpreter/index.html')
+                if self.tree:
+                    webbrowser.get('firefox').open_new_tab(
+                        'tree_html_interpreter/index.html')
+                else:
+                    webbrowser.get('firefox').open_new_tab(
+                        'graph_html_interpreter/index.html')
```

Comment:
This seems like an unnecessary code duplication. What about creating a variable `html_path`, and calling the `webbrowser.get('firefox').open_new_tab`  method only once and passing the variable as a parameter? I think that if you will wish to replace the `webbrowser.get('firefox').open_new_tab` by anything more sophisticated it would be easier. Again, please think about it.